src/theory/quantifiers/sygus/sygus_unif_rl.cpp

```diff
@@ -99,14 +235,14 @@ void SygusUnifRl::collectPoints(Node n)
 
 void SygusUnifRl::initializeConstructSol()
 {
-  if (d_hasRLemmas && d_rlemmas != d_prev_rlemmas)
+  if (d_rlemmas != d_prev_rlemmas)
```

Comment:
I don't think we need this code any more. You can also remove collectPoints above.